py_neuromodulation/nm_linelength.py

```diff
@@ -1,18 +1,18 @@
 import numpy as np
 from typing import Iterable
 
-from py_neuromodulation import nm_features_abc
+from py_neuromodulation.nm_features_abc import Feature
 
 
-class LineLength(nm_features_abc.Feature):
+class LineLength(Feature):
     def __init__(
         self, settings: dict, ch_names: Iterable[str], sfreq: float
     ) -> None:
         self.s = settings
         self.ch_names = ch_names
 
     @staticmethod
-    def get_line_length(x: np.ndarray) -> np.ndarray:
+    def get_line_length(x: np.ndarray) -> np.floating: # np.mean w/o axis argument
```

Comment:
That's the mean across all axes

Reply:
Sure, I was just clarifying why I changed the return type.